upstream.wrap

```diff
@@ -0,0 +1,6 @@
+[wrap-file]
+directory = fmt-7.0.1
+
+source_url = https://github.com/fmtlib/fmt/archive/7.0.1.tar.gz
+source_filename = fmt-7.0.1.tar.gz
+source_hash = ac335a4ca6beaebec4ddb2bc35b9ae960b576f3b64a410ff2c379780f0cd4948
```

Comment:
As of recently wraps should also have provide sections:

https://mesonbuild.com/Wrap-dependency-system-manual.html#provide-section